test/instrumentation/modules/hapi.js

```diff
@@ -13,9 +13,6 @@ var semver = require('semver')
 // hapi 17+ requires Node.js 8.9.0 or higher
 if (semver.lt(process.version, '8.9.0') && semver.gte(pkg.version, '17.0.0')) process.exit()
 
-// hapi does not work on Node.js 10 because of https://github.com/nodejs/node/issues/20516
-if (semver.gte(process.version, '10.0.0-rc')) process.exit()
```

Comment:
Rather than fully removing this, can we change it to a range to indicate that certain early versions of 10.x _are_ broken?

Reply:
Good idea. Fixed